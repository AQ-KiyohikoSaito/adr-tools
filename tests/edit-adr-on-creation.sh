export EDITOR="echo EDITOR"
export VISUAL=
adr new created with EDITOR environment variable set

export EDITOR=
export VISUAL="echo VISUAL"
adr new created with VISUAL environment variable set

export EDITOR="echo EDITOR"
export VISUAL="echo VISUAL"
adr new uses setting of VISUAL if both VISUAL and EDITOR are set
