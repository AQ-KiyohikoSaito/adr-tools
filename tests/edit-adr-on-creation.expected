+ export 'EDITOR=echo EDITOR'
+ EDITOR='echo EDITOR'
+ export VISUAL=
+ VISUAL=
+ adr new created with EDITOR environment variable set
EDITOR doc/adr/0001-created-with-editor-environment-variable-set.md
doc/adr/0001-created-with-editor-environment-variable-set.md
+ export EDITOR=
+ EDITOR=
+ export 'VISUAL=echo VISUAL'
+ VISUAL='echo VISUAL'
+ adr new created with VISUAL environment variable set
VISUAL doc/adr/0002-created-with-visual-environment-variable-set.md
doc/adr/0002-created-with-visual-environment-variable-set.md
+ export 'EDITOR=echo EDITOR'
+ EDITOR='echo EDITOR'
+ export 'VISUAL=echo VISUAL'
+ VISUAL='echo VISUAL'
+ adr new uses setting of VISUAL if both VISUAL and EDITOR are set
VISUAL doc/adr/0003-uses-setting-of-visual-if-both-visual-and-editor-are-set.md
doc/adr/0003-uses-setting-of-visual-if-both-visual-and-editor-are-set.md
